### DISCUSSION TITLE
SCA Support Planned for Pinax Stripe?

### DISCUSSION THREAD

**Discussion by trag-stripe** [pinax/pinax-stripe#647] [Q&A]
#### Issue Summary

Is Pinax-stripe actively maintained and are there plans to support PaymentIntents for European customers that have the [Strong Customer Authentication (SCA) requirements](https://stripe.com/docs/strong-customer-authentication) coming this fall?

I'm actively updating the list of [community-supported plugins](https://stripe.com/docs/libraries#third-party) and was curious about the health and maintenance of Pinax-stripe.

**Comment by paltman** [comment]
Working on this next weeK.

**Comment by graimon** [comment]
Any updates on this topic?

**Comment by maxipeli** [comment]
Hi, 
   is there an estimated date to release the support to SCA?
  
   thanks!

**Comment by ajlyakhov** [comment]
Can I manually and new webhook? I need to add "invoice.payment_intent.create" webhook to handle SCA scenario.

**Comment by paltman** [comment]
@trag-stripe see #645
